force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -163,9 +221,15 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer generateRandomNumber() {
         // Use Math class to generate number between 50 and 100
+        Integer randomNumber = 0;
+        randomNumber = (Math.random() * 100).intValue();
+
+        if (randomNumber < 50){
```

Comment:
You can also use the formula `(Math.random() * 50) + 50` to ensure the value is always between 50 and 100